app/layout.tsx

```diff
@@ -0,0 +1,28 @@
+'use client';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import React from 'react';
+import { Layout } from '../components/Layout';
+import '../styles/dashboard.css';
+import '../styles/globals.css';
+const queryClient = new QueryClient();
+
+export default function RootLayout({
+  // Layouts must accept a children prop.
+  // This will be populated with nested layouts or pages
+  children,
+}: {
+  children: React.ReactNode;
+}) {
```

Comment:
```suggestion
export const RootLayout = ({
  // Layouts must accept a children prop.
  // This will be populated with nested layouts or pages
  children,
}: {
  children: React.ReactNode;
}) => {
```

You can also get cleaner React Typescript typing by using the `React.FC<{
  children: React.ReactNode;
}>` typing

```suggestion
export const RootLayout: React.FC< {
  children: React.ReactNode;
}> = ({
  // Layouts must accept a children prop.
  // This will be populated with nested layouts or pages
  children,
}) => {
```